example/invoice-dx/app/src/lib.rs

```diff
@@ -13,21 +15,28 @@ use anonify_runtime::{
     state_type::*,
 };
 use crate::localstd::{
+    string::String,
     vec::Vec,
-    collections::BTreeMap,
 };
-use anonify_common::{UserAddress, OWNER_ADDRESS};
+use anonify_common::UserAddress;
 use codec::{Encode, Decode};
 
-pub const MAX_MEM_SIZE: usize = 100;
-pub const CIPHERTEXT_SIZE: usize = MAX_MEM_SIZE + 30;
-
-
+// pub const MAX_MEM_SIZE: usize = 100;
+// pub const CIPHERTEXT_SIZE: usize = MAX_MEM_SIZE + 30;
 
 impl_memory! {
-
+    (0, "Body", Text)
```

Comment:
prefer `"Invoice"` rather than `"Body"`

Reply:
fix it.
81a0f80f9d1f5faf0cdf2b474a385769310387af